src/storage/src/hummock/state_store.rs

```diff
@@ -301,12 +301,21 @@ impl HummockStorage {
                 continue;
             }
             let table_infos = prune_ssts(level.table_infos.iter(), &(key..=key));
```

Comment:
shall we need impletement the `prune` logic shrink inside to `prune_ssts` ?

Reply:
I think `prune_sst` is not necessary. Because binary-search is fast enough.